src/tools/mmaps_generator/MapBuilder.cpp

```diff
@@ -603,9 +603,7 @@ namespace MMAP
         config.borderSize = config.walkableRadius + 3;
         config.maxEdgeLen = VERTEX_PER_TILE + 1;        // anything bigger than tileSize
         config.walkableHeight = m_bigBaseUnit ? 3 : 6;
-        // a value >= 3|6 allows npcs to walk over some fences
-        // a value >= 4|8 allows npcs to walk over all fences
-        config.walkableClimb = m_bigBaseUnit ? 4 : 8;
+        config.walkableClimb = m_bigBaseUnit ? 3 : 6;
```

Comment:
maybe keep some comments that explain this?

Reply:
I doubt we'd ever have to edit those again, but yeah, I have reverted it for reference.